extension/ts/ui/toast.ts

```diff
@@ -1,87 +1,60 @@
-import {ToastType, ShowToastEvent} from "../types";
```

Comment:
UHHHH

I went through many iterations of this file. I cut so much that commit messages like "Added IDE support ... " don't event apply anymore (I no longer have that IDE support)

Reply:
AHAHAHAHA ty for your efforts, the world (this codebase) is a better place for it